tests/unit/__init__.py

```diff
@@ -0,0 +1 @@
+"""init package."""
```

Comment:
@vasylboliuk I meant the package name it contains. Like Unit pachage in this case